test/powershell/engine/ExecutionPolicy.GroupPolicy.Tests.ps1

```diff
@@ -0,0 +1,43 @@
+﻿##
+## PowerShell ExecutionPolicy GroupPolicy tests
+## These are Windows only tests
+##
+
+Describe "User group policy execution policy should work" -Tags 'Feature' {
+
+    BeforeAll {
+
+        $originalDefaultParameterValues = $PSDefaultParameterValues.Clone()
+        if (!$IsWindows)
+        {
+            $PSDefaultParameterValues["it:skip"] = $true
```

Comment:
This was a good addition. It just needs a small tweak. Non-Windows systems should not execute the rest of the BeforeAll and the end of the AfterAll as well.